lib/screens/home.dart

```diff
@@ -289,6 +289,51 @@ class _HomeScreenState extends State<HomeScreen> {
                         ),
                       ),
                     ),
+                    GestureDetector(
+                      onTap: () {
+                        Navigator.push(
+                          context,
+                          MaterialPageRoute(
+                            builder:(context) =>DSCLoyolaScreen()),
+                        );
+                      },
+                    child: Stack (
+                      children: <Widget> [
+                        Image.asset (
+                          "assets/orgs/dsc/cover.png"
+                        ),
+                        Expanded(
```

Comment:
Expanded will result to an error `incorrect use of parent data widget. expanded widgets must be placed inside flex widgets`

To solve this just simply remove expanded and use Container